src/components/ha-cards.html

```diff
@@ -95,7 +92,7 @@
   // groups: X
   // rest: 100
 
-  var PRIORITY = {
+  const PRIORITY = {
```

Comment:
this change from 'var' to 'const' seems to break the interface on safari (both macOS and iOS)
`ReferenceError: Can't find variable: PRIORITY`